scripts/versions.json

```
@@ -0,0 +1,24 @@
+{
+    "0.1.0": {
+        "date": "2024-04-16",
+        "native_sdk": {
```

```suggestion
        "firebase_sdk": {
```

nit: can we call this firebase_sdk?

---

Updated :)